Software/dist_windows/script.iss

```diff
@@ -67,11 +64,9 @@ Source: "content/prismatik-unhook.dll"; DestDir: "{app}"; MinVersion: 6.1.7600;
 Source: "content/prismatik-hooks32.dll"; DestDir: "{app}"; MinVersion: 6.1.7600; Flags: 32bit ignoreversion
 Source: "content/prismatik-unhook32.dll"; DestDir: "{app}"; MinVersion: 6.1.7600; Flags: 32bit ignoreversion
 Source: "content/platforms/*"; DestDir: "{app}\platforms"; Flags: ignoreversion
-Source: "content/translations/*.qm"; DestDir: "{app}\translations"; Flags: ignoreversion
 Source: "content/Plugins/*"; DestDir: "{#UserSettingsDirName}\Plugins"; Flags: onlyifdoesntexist createallsubdirs recursesubdirs
-; These files are needed only when building with MSVC - otherwise just comment them
-Source: "content/msvcr120.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/msvcp120.dll"; DestDir: "{app}"; Flags: ignoreversion
+; Visual Studio redistributable package
+Source: "content/vcredist_x64.exe"; DestDir: {tmp}; Flags: deleteafterinstall

```

Comment:
The problem with using the redist installers is the lack of a dependency between the entries in the control panel. Since we aren't using MSI installers we can't use the preferred version, merge modules, to ship the runtime. This is why i chose to include the dlls directly. It's also why the 32bit files are statically linked now - to avoid having to ship both runtimes

see http://blogs.msdn.com/b/vcblog/archive/2007/10/12/how-to-redistribute-the-visual-c-libraries-with-your-application.aspx